### DISCUSSION TITLE
How would I add this extra to any query received via google search ?  It complete disables "AI" support.

### DISCUSSION THREAD

**Discussion by tracure1337** [searxng/searxng#3503] [General]
https://tedium.co/2024/05/17/google-web-search-make-default/

Set the URL as https://www.google.com/search?q=%s&udm=14

This disables AI support completely.

**Comment by unixfox** [comment]
We have thought about it but the issue is that you won't get suggestions and answers anymore from google.

**Reply by tracure1337** [reply]
Is that a bad thing? I am only guessing but for  the searx community this is more of a feature? :-)

**Reply by virtadpt** [reply]
I'd call it a feature.  I'd have to do less post-processing to filter out the LLM-generated misinformation provided by Google in recent days.

**Reply by unixfox** [reply]
we thrive to welcome even the newbies in IT so that is not acceptable to remove these features.
we could allow to switch between general and WEB category using a setting but general category is going to stay by default until WEB is missing of two features explained.

**Reply by tracure1337** [reply]
I know this is a strong opinion. But newbies will probably helped the most from disabling this.
Read somewhere: "If my dog can bring me the newspaper & summarise it for me every day, but is wrong 20% of the time, I am fascinated by the dog, but still will read the newspaper myself". 

Every day I see newbies learning wrong stuff because AI is so cringe most of the time. 
We even have a term now for this. "Slop". Like spam but for "AI."

But at the end of the day, a option for powerusers to disable it is already fixing my problem and hopefully others too ;-)

**Comment by tracure1337** [comment]
https://github.com/return42/searxng/commit/f6ae9bf4d052d5663d7291d295b4d60917f455ca
Thanks! Much love ❤️

**Comment by return42** [comment]
FYI: I implemented a POC in https://github.com/searxng/searxng/pull/3505 .. please make experience with and report in the PR / thanks :+1:

**Comment by return42** [comment]
Due to lack of interest, I closed the DRAFT:

- https://github.com/searxng/searxng/pull/3505
